### PR TITLE
Text in Basic Data Structures lesson is better clarified.

### LINKED ISSUE #40412: Text in Basic Data Structures lesson can be better clarified.
**Describe your problem and how to reproduce it:**
The lesson Basic Data Structures: Add Items Using splice() has an introductory text which has an implication that allows for misinterpretation of the `splice()` function. We discussed this in the FCC Discord under the software development channel.

The text: "We then pass the following to splice(). The index at which to begin deleting elements (3), the number of elements to be deleted (1), and the elements (13, 14) to be inserted at that same index" implies we insert two elements in the same index. This is (to my understanding) incorrect within the context of the given example. 

The text can be improved to clarify that we _**start**_ to insert the elements from that index. 

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-data-structures/add-items-using-splice

My apologies if I glossed over something or misinterpreted what was written in the introduction text.

### PR DESCRIPTION
**Pull request by jagadeeshg16**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40412

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/add-items-using-splice.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ console.log(numbers);
 // returns [ 10, 11, 12, 13, 14, 15 ]
 ```
 
-Here we begin with an array of numbers. We then pass the following to `splice()`. The index at which to begin deleting elements (3), the number of elements to be deleted (1), and the elements (13, 14) to be inserted at that same index. Note that there can be any number of elements (separated by commas) following `amountToDelete`, each of which gets inserted.
+Here we begin with an array of numbers. We then pass the following to splice(). The index at which to begin deleting elements (3), the number of elements to be deleted (1), and the remaining arguments (13, 14) will be inserted starting at that same index. Note that there can be any number of elements (separated by commas) following `amountToDelete`, each of which gets inserted.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/add-items-using-splice.md:23]:
```suggestion
Here, we begin with an array of numbers. Then, we pass the following to `splice()`: The index at which to begin deleting elements (3), the number of elements to be deleted (1), and the remaining arguments (13, 14) will be inserted starting at that same index. Note that there can be any number of elements (separated by commas) following `amountToDelete`, each of which gets inserted.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Nitpick

**Review by ShaunSHamilton — APPROVED** [review]
This LTGM 👍 

Thank you, for this PR, and congrats on your first PR to this repo. We look forward to future contributions.

Note: Failing test is not related to this PR

**Comment by GJ-16** [comment]
@ShaunSHamilton Thank you.
What is LTGM?

**Comment by ieahleen** [comment]
@GJ-16 it's "Looks Good To Me"

**Comment by ShaunSHamilton** [comment]
@GJ-16 Haha, it was a typo on my  part. As @ieahleen said, LGTM

**Comment by GJ-16** [comment]
I didn't know, I Googled and google said "Long Term Growth Model". So I was confused.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that we are currently in a code freeze, so this cannot be merged until the freeze is lifted. 🙂

**Comment by GJ-16** [comment]
@nhcarrigan Thanks so much, Thanks for the information mate.

**Comment by RandellDawson** [comment]
Congrats @J-16 on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**